src/Apps/__stories__/OrderApp.story.tsx

```diff
@@ -65,6 +65,30 @@ storiesOf("Apps/Order Page/Buy Now/Review", module).add("Review", () => (
   />
 ))
 
+storiesOf("Apps/Order Page/Buy Now/NewPayment", module)
+  .add("With 'Ship'", () => (
+    <Router
+      initialRoute="/orders/123/payment/new"
+      order={{
+        ...OfferOrderWithShippingDetails,
+        state: "SUBMITTED",
+        stateExpiresAt: null,
+        lastOffer: null,
+      }}
+    />
+  ))
+  .add("With 'Pickup'", () => (
+    <Router
+      initialRoute="/orders/123/payment/new"
+      order={{
+        ...BuyOrderPickup,
```

Comment:
I am pretty sure (will check tomorrow with Stephen) that we only want to allow this page to be accessed if you have an offer-type order. If so we can remove these two stories (otherwise we should update the above fixture to not be `OfferOrderWithShippingDetails`).

Reply:
done